src/test/UserTest/UserControllerIntegrationTest.java

```diff
@@ -50,6 +50,42 @@ public void loginUserWithNoUsernameTest() {
     assertThat(actualResponseJSON.getString("status")).isEqualTo("AUTH_FAILURE");
   }
 
+  @Test
+  public void testUserEncryption() {
+    TestUtils.login("adminBSM", "adminBSM");
+    JSONObject body = new JSONObject();
+    body.put("username", "adminBSM");
+
+    HttpResponse<String> actualResponse =
+        Unirest.get(TestUtils.getServerUrl() + "/get-user-info").asString();
+    JSONObject actualResponseJson = TestUtils.responseStringToJSON(actualResponse.getBody());
+    assertThat(actualResponseJson.get("city").toString()).isEqualTo("Philadelphia");
+    assertThat(actualResponseJson.get("firstName").toString()).isEqualTo("Mike");
+    assertThat(actualResponseJson.get("lastName").toString()).isEqualTo("Dahl");
+    assertThat(actualResponseJson.get("zipcode").toString()).isEqualTo("19104");
+    assertThat(actualResponseJson.get("phone").toString()).isEqualTo("1234567890");
+    assertThat(actualResponseJson.get("address").toString()).isEqualTo("311 Broad Street");
+    assertThat(actualResponseJson.get("birthDate").toString()).isEqualTo("06-16-1960");
```

Comment:
see below

Reply:
Yes, we will make sure to test these. Would your team be able to work on fixing the failing test cases, or should I do it (both Abhishek and Austin who wrote these left). I can if I need.

Reply:
Actually, I will make sure all the tests pass